### ISSUE TITLE
SA0 is undefined

### ISSUE THREAD

**Issue by pvmm** [MartinezTorres/sdcc_msx#1]
I think something is missing. When I try  to compile the `hello_world` demo I get
```
main.c:76: error 20: Undefined identifier 'SA0'
```
or maybe some instruction is missing in `README.md`?

**Comment by MartinezTorres** [comment]
That's true! I modified extensively the library for my last msx-dev submission, and forgot to test the demos. I will check this ASAP.

**Comment by MartinezTorres** [comment]
The hello world example should compile now. Ensure that you are using at least version 3.9 of sdcc, and that you place the correct path to it in the Makefile. The test_audio and test_gif demos are still not working (the audio part was extensively modified for the anchors aweigh), but I am working on it.
